WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShippingCreateLabelsView.swift

```diff
@@ -32,8 +45,131 @@ struct WooShippingCreateLabelsView: View {
 
                     WooShippingPackageAndRatePlaceholder()
                 }
+                .padding()
+            }
+            .safeAreaInset(edge: .bottom) {
+                ExpandableBottomSheet(onChangeOfExpansion: { isExpanded in
+                    isShipmentDetailsExpanded = isExpanded
+                }) {
+                    if isShipmentDetailsExpanded {
+                        VStack(spacing: Layout.bottomSheetSpacing) {
+                            Toggle(Localization.BottomSheet.markComplete, isOn: .constant(false)) // TODO: 14044 - Handle this toggle setting
```

Comment:
We should store this setting in the view model, to use when the purchase action is done. We need to mark the order as complete separate from the actual label purchase — there's a method `OrderDetailsViewModel.markCompleted(flow:)` we can use for that.